eventarc_gke/eventarc/main.tf

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
Can you please change the directory structure so that eventarc is the parent and then each sample should have a descriptive directory name under the top-level eventarc directory.

For example:

eventarc/descriptive_samplename_1/main.tf
eventarc/descriptive_sample_name_2/main.tf

For more info, see [go/terraform-docs-samples-repo-cleanup-actions](http://goto.google.com/terraform-docs-samples-repo-cleanup-actions)

Reply:
Done, thank you!